src/configuration/routes.md

```diff
@@ -196,6 +196,38 @@ function get_domain_for(string $id) {
 
 That can be used, for example, for inbound request whitelisting, a feature of many PHP frameworks.
 
+## Route attributes
+
+Route attributes are an arbitrary key/value pair attached to a route.  This metadata does not have any impact on Platform.sh, but will be available in the route definition structure in `$PLATFORM_ROUTES`.
+
+```
+"http://{default}/":
+    type: upstream
+    upstream: "app:http"
+    attributes:
+        "foo": "bar"
+```
+
+Will appear in the routes data in PHP like so:
+
+```text
```

Comment:
Maybe you should add a snippet showing something like:

```
function route_attributes() {
    foreach ($_ENV["PLATFORM_ROUTES"] as $route_key => $route) {
        if ($route["original_route"] === $_REQUEST["HTTP_X_ORIGINAL_ROUTE"]) {
            return $route["attributes"];
        }
    }
}

$current_attributes = route_attributes();
```

??

Reply:
It's actually a bit more complex than that; that logic may get you a redirect.  We have "how to mess with routes" documented elsewhere, mainly in config reader.

Reply:
Then probably mention it? After reading this I'm interested in figuring out how I can mess with routes to get the current one, so a link would be handy.